File: TaccomStrike.Library.Data/ApiEntities/GetGameLobby.cs
using System;
using System.Collections.Generic;
using System.Text;
using TaccomStrike.Library.Data.Enums;
using TaccomStrike.Library.Data.ViewModel;

namespace TaccomStrike.Library.Data.ApiEntities
{
	public class GetGameLobby
	{
		public string GameLobbyName { get; set; }

		public long GameLobbyID { get; set; }

		public int MaxRoomLimit { get; set; }

		public GameMode GameMode { get; set; }

		public GetUser Host { get; set; }

		public List<GetUser> Players { get; set; }
	}
}

File: TaccomStrike.Web.UI/src/models/rest/getgamelobby.ts
import { GetUser } from "./getuser";

export class GetGameLobby 
{
    gameLobbyName: string;
    gameLobbyID: number;
    maxRoomLimit: number;
    host: GetUser;
    gameMode: GameMode;
    players: GetUser[];
}

export enum GameMode 
{
    Casual=1, Competitive=2
}